### DISCUSSION TITLE
Opposition of Jupiter during October 1999

### DISCUSSION THREAD

**Discussion by HeySmarty** [skyfielders/python-skyfield#866] [Q&A]
I understand that the Jupiter was at opposition on SAT, 23 OCT 1999 AT 18:56 UTC.

I used the skyfield example Code for Mars, and modified for the Jupiter as follows:

> #Are planets separated by 0° at conjunction and 180° at opposition?
> from skyfield.api import load
> from skyfield.framelib import ecliptic_frame
> 
> ts = load.timescale()
> eph = load('de421.bsp')
> sun, jupiter = eph['sun'], eph['jupiter barycenter']
> 
> t = ts.utc(1999, 10, 23, 18, 56, 0)
> e = earth.at(t)
> s = e.observe(sun).apparent()
> m = e.observe(jupiter).apparent()
> 
> print('Latitude Longitude')
> lat, lon, distance = s.frame_latlon(ecliptic_frame)
> print('Sun  {:.5f}° {:.5f}°'.format(lat.degrees, lon.degrees))
> 
> lat, lon, distance = m.frame_latlon(ecliptic_frame)
> print('Jupiter {:.5f}° {:.5f}°'.format(lat.degrees, lon.degrees))
> 
> print('Separation: {:.5f}°'.format(m.separation_from(s).degrees))

I got the following output:

> Latitude Longitude
> Sun  -0.00000° 209.91967°
> Jupiter -1.53915° 29.92584°
> Separation: 178.46084°

I hope the Code is correctly modified, but i do not understand how the latitude for the Sun is -0.00000°, nor its meaning.

I would like some assistance to understand the numbers.

Thanks in advance.

**Comment by brandon-rhodes** [comment]
> I hope the Code is correctly modified, but i do not understand how the latitude for the Sun is -0.00000°, nor its meaning.

Could you provide a bit more detail about your confusion — is your question about (a) why a negative number has a minus sign in front, or (b) about the official astronomy definition of "ecliptic latitude", or (c) are you wondering why the Sun's latitude is so close to zero?
